locale/de.properties

```diff
@@ -0,0 +1,69 @@
+activate_on_site = Klicke hier um den Privacy Badger auf dieser Seite zu aktiveren.
+click_badger_activate_on_site = Klicke auf das Dachs-Symbol um den Privacy Badger auf dieser Seite zu aktivieren.
+cookie_prefs_warning = <p id="cookiePrefsWarning">
+Achtung: Der Privacy Badger Alpha arbeitet vielleicht nicht wie erwartet mit
```

Comment:
Yes it sounds a bit weird in both ways, but as the text seems to be a "personal" introduction of the badger I think it is okay to say "the".